tests/screenshot.spec.ts

```diff
@@ -183,46 +211,7 @@ browserTest.describe('element screenshot', () => {
     await context.close();
   });
 
-  browserTest('should restore default viewport after fullPage screenshot', async ({ browser }) => {
-    const context = await browser.newContext({ viewport: { width: 456, height: 789 } });
-    const page = await context.newPage();
-    await verifyViewport(page, 456, 789);
-    const screenshot = await page.screenshot({ fullPage: true });
-    expect(screenshot).toBeInstanceOf(Buffer);
-    await verifyViewport(page, 456, 789);
-    await context.close();
-  });
-
-  browserTest('should restore viewport after page screenshot and exception', async ({ browser, server, mode }) => {
```

Comment:
Same here and below, old tests should stay.

Reply:
Will do.